[2022-05-26 14:53:09,666] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-05-26T14:53:08.315533+00:00 [queued]>
[2022-05-26 14:53:09,674] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-05-26T14:53:08.315533+00:00 [queued]>
[2022-05-26 14:53:09,674] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 14:53:09,674] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-05-26 14:53:09,674] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-05-26 14:53:09,689] {taskinstance.py:1378} INFO - Executing <Task(BashOperator): also_run_this> on 2022-05-26 14:53:08.315533+00:00
[2022-05-26 14:53:09,694] {standard_task_runner.py:52} INFO - Started process 525 to run task
[2022-05-26 14:53:09,697] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'manual__2022-05-26T14:53:08.315533+00:00', '--job-id', '15', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpju1dv8cz', '--error-file', '/tmp/tmpo1et3axk']
[2022-05-26 14:53:09,697] {standard_task_runner.py:80} INFO - Job 15: Subtask also_run_this
[2022-05-26 14:53:09,741] {task_command.py:370} INFO - Running <TaskInstance: example_bash_operator.also_run_this manual__2022-05-26T14:53:08.315533+00:00 [running]> on host b7bcf2b221fa
[2022-05-26 14:53:09,794] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-05-26T14:53:08.315533+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-26T14:53:08.315533+00:00
[2022-05-26 14:53:09,794] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-26 14:53:09,795] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=manual__2022-05-26T14:53:08.315533+00:00 | dag_run=<DagRun example_bash_operator @ 2022-05-26 14:53:08.315533+00:00: manual__2022-05-26T14:53:08.315533+00:00, externally triggered: True>"']
[2022-05-26 14:53:09,801] {subprocess.py:85} INFO - Output:
[2022-05-26 14:53:09,802] {subprocess.py:92} INFO - run_id=manual__2022-05-26T14:53:08.315533+00:00 | dag_run=<DagRun example_bash_operator @ 2022-05-26 14:53:08.315533+00:00: manual__2022-05-26T14:53:08.315533+00:00, externally triggered: True>
[2022-05-26 14:53:09,802] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-26 14:53:09,821] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220526T145308, start_date=20220526T145309, end_date=20220526T145309
[2022-05-26 14:53:09,868] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-26 14:53:09,887] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
